(feat#22) Partner Organization Query

santyssk opened the pull request.

dickwyn commented:
@santyssk Include a description of your changes and in the PR title, follow what is in the CONTRIBUTING guide https://github.com/asucis/v3/blob/dev/CONTRIBUTING.md

dickwyn commented on src/pages/partnerorg.js:
```diff
@@ -0,0 +1,28 @@
+          <p>{post.node.frontmatter.link}</p>
+          <p>{post.node.frontmatter.description}</p>
+        </div>
+      ))}
```
Is it possible to put this logic before returning the orgs.

So when a user calls this function, they can do data[0].title directly and not data.allMarkdownRemark.edges[0].node.frontmatter.title

santyssk replied:
Will check that

dickwyn replied:
So the user can do something like that.

 const data = getPartnerOrganization();

<p>{data[0].title}</p>
<p>{data[0].email}</p>
<p>{data[0].photo}</p>
<p>{data[0].link}</p>
<p>{data[0].description}</p>

santyssk commented:
feat(#22 ): Partner Organization Query

Added query for getting data of partner organizations in "src/hooks/get-partner-organization.js"
Created a new page for partner organizations "src/pages/partnerorg.js"
Added a link to partner organization from the first page in "src/pages/index.js"
